Tasks/between.js

```diff
@@ -1,21 +1,10 @@
 // Extract substring between prefix and suffix
+'use strict'
 
-getvaluebetween = (str, p, s) => {
-  i = str.indexOf(p);
-  if (i === -1) return '';
-  else {
-    k = i + p.length;
-    str = str.substring(k);
-    if (s) {
-      i = str.indexOf(s);
-      if (i === -1) {
-        return '';
-      } else {
-        str = str.substring(0, i);
-      }
-    }
-  }
-  return str;
+const getvaluebetween = (fullString, startString, endString) => {
```

Comment:
```suggestion
const getValueBetween = (fullString, startString, endString) => {
```
code style not visible